src/encoded/cart_view.py

```diff
@@ -30,7 +30,7 @@ def get_userid(request):
 def get_cart_objects_by_user(request, userid, blocked_statuses=[]):
     request.datastore = 'database'
     return [
-        request.embed(request.resource_path(v), '@@object')
+        dict(v.properties, **{'@id': request.resource_path(v)})
```

Comment:
I'm not sure what this line does, maybe @caseylitton can review?

Reply:
`request.embed` takes a very long time to execute, and all we need is the `@id`. So this line gets the `@id` of the cart object and puts it into the `@id` property of the cart item. This is to get around that the cart item doesn’t have an @id nor uuid, so `request.embed` adds it but is extremely slow. This way we get the `@id` quickly

Reply:
With four carts with hundreds of items each, this was taking 5.6 seconds with request.embed. With this change, it took 283ms

Reply:
Oh wow, 5.6 seconds each! 283 ms is much more reasonable.